src/main.rs

```diff
@@ -135,6 +135,15 @@ fn main() {
             },
             "people_now_present".into(),
         )
+        .add_sensor(
+            PeopleNowPresentSensorTemplate {
+                location: Some("Cyberspace".into()),
+                name: None,
+                description: Some("Our virtual hackerspace meeting through Jitsi (because of COVID)".into()),
```

Comment:
The sensor description is a bit long:
![photo6041858683333358090](https://user-images.githubusercontent.com/1435346/77568659-426ad180-6ec9-11ea-847e-41afa132ee5a.jpg)

Reply:
That's a UI bug, please report it to the myhackerspace project :slightly_smiling_face:

Reply:
Done https://github.com/fixme-lausanne/MyHackerspace/issues/78